docs/standard/native-interop/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Native interoperability
```

Comment:
"interoperability" should be capitalized if it is a title - throughout.

Reply:
That's not correct. Microsoft's style guide asks for sentence case:
https://docs.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings

Reply:
Oh - then we are very inconsistent at https://docs.microsoft.com/en-us/dotnet/standard/

Thanks for pointing to that document!

Reply:
Yes we are 😢 

It's in our backlog: https://github.com/dotnet/docs/issues/808

We've been fixing as we go